pkg/graveler/combined_iterator.go

```diff
@@ -2,8 +2,6 @@ package graveler
 
 import (
 	"bytes"
-
-	"github.com/treeverse/lakefs/pkg/logging"
 )
 
 // CombinedIterator iterates over two listing iterators,
```

Comment:
I think documentation is crucial here. Moreover it still says "CombinedIterator iterates over two listing iterators", while this is "technically" true, it is still confusing

Reply:
Solved in #3688